src/components/date-time-picker/date-time-picker.spec.ts

```diff
@@ -94,6 +97,28 @@ describe('Date Time Picker', () => {
         expect(getSelected().innerHTML).toBe(' 31 ');
     });
 
+    it('should default to local timezone', async() => {
+        component.showTime = true;
+        component.showTimezone = true;
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        expect(getGMT().getAttribute('ng-reflect-value')).toBe('GMT');
```

Comment:
As above, I think inspecting the input value is preferable. Also, is this still going to work in the summer, or if our international colleagues run the tests?